<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBNetworkFluidEditWidget</class>
 <widget class="QWidget" name="SVDBNetworkFluidEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="labelDisplayName">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName"/>
   </item>
   <item row="2" column="2">
    <widget class="QtExt::ColorButton" name="pushButtonColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxModelParameters">
     <property name="title">
      <string>Fluid properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Density [kg/m3]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditDensity"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Spec. heat capacity [J/kg K]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditHeatCapacity"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Thermal conductivity [W/mK]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditThermalConductivity"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidgetViscosity"/>
      </item>
      <item row="3" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="toolButtonAddViscosityPoint">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/SIM-VICUS.qrc">
            <normaloff>:/gfx/actions/24x24/plus.png</normaloff>:/gfx/actions/24x24/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonRemoveViscosityPoint">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/SIM-VICUS.qrc">
            <normaloff>:/gfx/actions/24x24/minus.png</normaloff>:/gfx/actions/24x24/minus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
   <slots>
    <signal>editingFinishedSuccessfully()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/SIM-VICUS.qrc"/>
 </resources>
 <connections/>
</ui>
